### ISSUE TITLE
add support to mathjax

### ISSUE THREAD

**Issue by zexiangliu** [iamcco/markdown-preview.nvim#198]
Hello, I really like this plugin and am wondering if it is possible to add support to Mathjax, such as a setting option that allows a switch between Ketax and Mathjax. I noticed that the plugin `mathjax-support-for-mkdp` used to do this job, but seems not working or has conflicts with Ketax.

I know Ketax is better than Mathjax in terms of the rendering time. But Mathjax has the following benefits that Ketax does not have yet:
- support a large amount of latex macros such as `align` and `equation`.
- support equation numbering and reference, that is `label` and `eqref`.
- the rendered math looks prettier. 

Those features come to be very convenient when one is writing math blogs. 
Also, I think issues like #114 can be solved by adding support to Mathjax.

**Comment by mscott99** [comment]
Additionally, it would be great if there was a way to support user-defined latex macros. I only know that this is done in obsidian, they allow a global sty file that is loaded by mathjax. https://github.com/wei2912/obsidian-latex
